src/github.com/getlantern/lantern-android/app/src/go/Go.java

```diff
@@ -0,0 +1,47 @@
+package go;
```

Comment:
The lantern-android package [hasn't been accepted yet][1]. I think we should wait until its PR gets completed before adding it to the source.

Would it be possible to `git revert` the commit that added this package instead of using `git rm`? I think it would be safer for the lantern-android's [pending PR][1].

[1]: https://github.com/getlantern/flashlight-build/pull/36